swagger.yaml

```diff
@@ -3404,6 +3404,16 @@ paths:
           type: string
           format: uuid
           required: false
+        - in: query
+          name: offset
+          description: Number of records to skip before starting to return the records.
+          type: integer
+          required: false
+        - in: query
+          name: limit
```

Comment:
question - limit == number of records per page?

Reply:
another question - are we going to define a default number for the limit? I mean if the user will call the event API without setting the offset and the limit - will he get the same data as we have today with all the events(even if we have 1 million events)?

Reply:
> question - limit == number of records per page?
Maximum number of records per page yes @gamli75

Reply:
> another question - are we going to define a default number for the limit? I mean if the user will call the event API without setting the offset and the limit - will he get the same data as we have today with all the events(even if we have 1 million events)?

Yes, currently the logic that seems most reasonable for me and the UI as well AFAIK is:

- `limit` not set -> no limit (limit = -1)
- `limit` is set but is invalid value (limit < -1 or limit = 0) -> limit = < some default value >
- `offset` not set -> start from the first event (offset = 0)
- `offset` is set but invalid (offset < 0) -> start from the first event (offset = 0)
- `offset` is set but larger than the total number of matching records in the DB -> no results will be returned

WDYT? @gamli75